apply.go

```diff
@@ -47,13 +47,26 @@ func apply(c *cli.Context) error {
 		translateNetworkToIds(&expected, swarm, stack.Name)
 		current := getSwarmServicesSpecForStack(services)
 
+		cyan := color.New(color.FgCyan)
+		for name, cs := range current {
+			if _, found := targetMap[cs.Spec.Name]; len(targetMap) == 0 || found {
+				if _, found := expected[name]; !found {
+					// service exists but it's not expected, need to delete it
+					cyan.Printf("Removing service %s\n", name)
```

Comment:
Removing and deleting are both in Cyan?. Wouldn't it make sense to use Red for deletion?